e2e/regression/regression_test.go

```diff
@@ -89,11 +97,128 @@ func TestRegression(t *testing.T) {
 			require.NoError(c.WaitFor(ctx, kubeclient.Ready, kubeclient.Deployment{}, ct.Namespace, deploymentName))
 		})
 	}
+
+	t.Run("http-proxy", func(t *testing.T) { testHTTPProxy(t, ct) })
+}
+
+func testHTTPProxy(t *testing.T, ct *contrasttest.ContrastTest) {
```

Comment:
> I wonder how this change interferes with the `get-logs` script, though.

The e2e test setup expects exactly one namespace per run. So there should only be one `func Test...` per e2e test.

This happened:
* first `func Test...` runs
* `get-logs start` reads the first namespace name
* second `func Test...` runs and overwrites the namespace file
* `get-logs download` reads the second namespace name
* `grep log-collector` causes pipefail

(namespace cleanup is affected similarly)

Reply:
Thanks for debugging this, I forgot about the generated namespace.